app/src/main/java/org/dhis2/usescases/teiDashboard/TeiDashboardMobileActivity.java

```diff
@@ -96,33 +98,63 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
 
         binding.tabLayout.setupWithViewPager(binding.teiPager);
         binding.tabLayout.setTabMode(TabLayout.MODE_SCROLLABLE);
+        binding.tabLayout.addOnTabSelectedListener(new TabLayout.OnTabSelectedListener() {
+            @Override
+            public void onTabSelected(TabLayout.Tab tab) {
+                if (tab.getPosition() == binding.tabLayout.getTabCount() - 1) {
```

Comment:
I think it could me more readable to have a variable NOTES_TAB with binding.tabLayout.getTabCount() - 1 value